### ISSUE TITLE
NVDA's say all function doesn't work properly with speechHistory addon if "allow skim reading in say all is enabled

### ISSUE THREAD

**Issue by Surveyor123** [jscholes/nvda-speech-history#44]
To reproduce:
1. Be sure you've enabled the speech history review and coppying addon and checked "allow skim reading in say all" option under NVDA's keyboard settings.
2. Open a document and give say all command.
3. press any key such as control that will stop NVDA's speech.
4. Press up or down arrow or if the document has more than one heading or link press the single key for next or previous one.
5. You'll notice say all continues from the position you've jumped.
Expected behavior:
After stopping NVDA speech during say all, say all session should be closed until the next say all command.

**Comment by Surveyor123** [comment]
First, Thank you for the update. However, after updating to 2026.1, the patch I've implemented to solve the sayAll issue on my local copy has broken. fortunately, I've changed mySpeak as below and it seems the issue has disappeared again.
	def mySpeak(self, sequence, *args, **kwargs):
		try:
			if not isinstance(sequence, list):
				return self.oldSpeak(sequence, *args, **kwargs)
			
			import speech.commands
			for item in sequence:
				if isinstance(item, (speech.commands.IndexCommand, speech.commands.CallbackCommand)):
					return self.oldSpeak(sequence, *args, **kwargs)
		except Exception:
			return self.oldSpeak(sequence, *args, **kwargs)

		self.oldSpeak(sequence, *args, **kwargs)
		
		try:
			text = self.getSequenceText(sequence)
			if text and text.strip():
				queueFunction(eventQueue, self.append_to_history, sequence)
		except Exception:
			pass
